src/psi-behavior.scm

```diff
@@ -76,7 +80,6 @@
 		(DefinedPredicate "Did someone leave?")))
 	(DefinedSchemaNode "Someone left action")
 	face-demand-satisfied (stv 1 1) face-demand)
-
```

Comment:
please don't remove blank lines, unless its actually appropriate. This one looks like a random accident  with the editor.